src/main.coffee

```diff
@@ -7,6 +7,7 @@ path      = require 'path'
 tmp       = require 'tmp'
 session   = require('electron').session
 log       = require('bog');
+windowState = require('./ui/models/windowstate')
```

Comment:
This breaks the separation we have between main.coffee and UI

I'd prefer if we used the communication between the main process and UI instead of sharing code.

Maybe use the event 'ready-to-show' on the UI to reposition the window? Or whatever infrastructure was used befoer